packages/web-twig/src/Resources/components/Toast/ToastBar.twig

```diff
@@ -0,0 +1,60 @@
+{# API #}
+{%- set props = props | default([]) -%}
+{%- set _closeLabel = props.closeLabel | default('Close') -%}
+{%- set _color = props.color | default('inverted') -%}
+{%- set _hasIcon = props.hasIcon | default(false) -%}
+{%- set _id = props.id | default(null) -%}
+{%- set _isDismissible = props.isDismissible | default(false) -%}
+{%- set _isOpen = props.isOpen ?? true -%}
+
+{# Class names #}
+{%- set _rootClassName = _spiritClassPrefix ~ 'ToastBar' -%}
+{%- set _rootColorClassName = _spiritClassPrefix ~ 'ToastBar--' ~ _color -%}
+{%- set _rootDismissibleClassName = _isDismissible == 'true' ? _spiritClassPrefix ~ 'ToastBar--dismissible' : null -%}
+{%- set _contentClassName = _spiritClassPrefix ~ 'ToastBar__content' -%}
+{%- set _messageClassName = _spiritClassPrefix ~ 'ToastBar__message' -%}
+{%- set _isOpenClassName = _isOpen ? 'is-open' : 'is-hidden' -%}
+
+{# Attributes #}
+{%- set _idAttr = _id ? 'id="' ~ _id | escape('html_attr') ~ '"' : null -%}
+
+{# Miscellaneous #}
+{%- set _styleProps = useStyleProps(props) -%}
+{%- set _classNames = [ _rootClassName, _rootColorClassName, _rootDismissibleClassName, _isOpenClassName, _styleProps.className ] -%}
+{%- set _iconNameByColor = 'info' -%}
+{% if _color == 'success' %}
+    {%- set _iconNameByColor = 'check-plain' -%}
+{% elseif _color == 'warning' %}
+    {%- set _iconNameByColor = 'warning' -%}
+{% elseif _color == 'danger' %}
+    {%- set _iconNameByColor = 'danger' -%}
+{% endif %}
+
+<div
+    {{ mainProps(props) }}
+    {{ styleProp(_styleProps) }}
+    {{ classProp(_classNames) }}
+    {{ _idAttr | raw }}
+>
+    <div class="{{ _contentClassName }}">
+        {% if _hasIcon %}
+            <Icon name="{{ _iconNameByColor }}" />
+        {% endif %}
+        <div class="{{ _messageClassName }}">
+            {% block content %}{% endblock %}
+        </div>
+    </div>
+    {% if _isDismissible == 'true' %}
```

Comment:
I copied this from `Tooltip`. Does anyone know why there is the string comparison instead of plain `if _isDismissible`…?

Reply:
@literat? 

TBH I don't know. Maybe because `<toastbar isDismissible="false" />`? But you shouldn't use it like this, so I don't know why.

Reply:
Hmm, 🤔 I do not know 🤷 @adamkudrna is right in the usage. We should use the Boolean internally. So I am not quite sure why I wrote it like this, because there is no usage of the `<toastbar isDismissible="false" />`.

I think we should fix in in both places.